### DISCUSSION TITLE
Project export/import

### DISCUSSION THREAD

**Discussion by nunopoco** [illacloud/illa-builder#1886] [Ideas]
Hi, as mentioned by Owen, I'm requesting that you consider adding project export and import into Illa Cloud.
This helps us to be able to develop on cloud, export the project and then import into customer's self-host.
Thanks for considering adding this to the roadmap.

**Comment by robomotionpro** [comment]
That's an awesome idea, upvoted.

**Reply by lloydbh74** [reply]
Very sensible suggestion. I have several clients who would prefer a self-hosted solution for their internal applications.
